src/settings.js

```diff
@@ -17,8 +18,8 @@ export default class Settings {
   }
 
   get selections() {
-    const bench = context.sharedStorage.get(BENCH, 1)
-    const bin = context.sharedStorage.get(BIN, 1)
+    const bench = context.sharedStorage.get(BENCH, 0)
+    const bin = context.sharedStorage.get(BIN, 0)
```

Comment:
I had this set to `0` but for some scenarios I was getting an empty value...

Reply:
Do you recall the scenarios? This would only happen if `this.benches` was empty or contained an invalid value for some reason.

Reply:
I think it was Build Your Own Six Flags Belgium

Reply:
I think that's an unrelated issue. `benches`/`bins` seems to get corrupted when switching scenarios.
![image](https://user-images.githubusercontent.com/812086/94350299-d1d07900-0001-11eb-9c8e-538a99b29398.png)

Reply:
damn, gotta figure that out so settings can be preserved.

Reply:
The problem is that `benches`/`bins` only gets initialized once. However, which additions are available depends on the scenario.

Reply:
I created #29 to fix that. `0` seems to correctly work in all instances now.